src/types/onyx/PersonalDetails.ts

```diff
@@ -81,8 +81,10 @@ type PersonalDetails = OnyxCommon.OfflineFeedback<'avatar' | 'originalFileName'>
     status?: Status;
 };
 
-type PersonalDetailsList = Record<string, PersonalDetails | null>;
+type PersonalDetailsWithOfflineFeedback = OnyxCommon.OnyxItemWithOfflineFeedback<PersonalDetails, keyof PersonalDetails>;
 
-export default PersonalDetails;
+type PersonalDetailsList = Record<string, PersonalDetailsWithOfflineFeedback | null>;
+
+export default PersonalDetailsWithOfflineFeedback;
```

Comment:
```suggestion
export default PersonalDetails;
```

Same, let's match with file name